test/routers/handlers/new-submission/newSubmissionHandler.int.ts

```diff
@@ -0,0 +1,49 @@
+// import { Resource } from "@companieshouse/api-sdk-node";
+// import { Transaction } from "@companieshouse/api-sdk-node/dist/services/transaction/types";
+import { HttpStatusCode } from "axios";
```

Comment:
Comments can be deleted here

```suggestion
import { HttpStatusCode } from "axios";
```